spec/nexpose_upload_spec.rb

```diff
@@ -86,25 +86,20 @@
         expect(args[:node].label).to eq("Nexpose Scan Summary")
       end.once
 
-      expect(@content_service).to receive(:create_node).with(hash_including label: "1.1.1.1", type: :host).once
+      expect(@content_service).to receive(:create_node).with(
+        hash_including label: "1.1.1.1", type: :host
+      ).twice
+
       expect(@content_service).to receive(:create_note) do |args|
-        expect(args[:text]).to include("#[Host]#\n1.1.1.1")
+        expect(args[:text]).to include("#[Title]#\n1.1.1.1")
         expect(args[:node].label).to eq("1.1.1.1")
       end.once
 
-      expect(@content_service).to receive(:create_node) do |args|
```

Comment:
Why are we removing this part?

Reply:
Looks like creating the Definitions node was removed before: https://github.com/dradis/dradis-nexpose/pull/11 (https://github.com/dradis/dradis-nexpose/pull/11/commits/fc99612b2853ab216dd66ad2d40b72ec95925e4e) but the spec was still here (failing)